### PR TITLE
Instructions to report security vulnerabilities

### PR DESCRIPTION
**Pull request by baijum**
Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,22 +1,5 @@
...
-## Supported Versions
-
-Information about supported Kubernetes versions can be found on the
-[Kubernetes version and version skew support policy] page on the Kubernetes website.
-
-[kubernetes-security-announce]: https://groups.google.com/forum/#!forum/kubernetes-security-announce
-[kubernetes-security-announce-rss]: https://groups.google.com/forum/feed/kubernetes-security-announce/msgs/rss_v2_0.xml?num=50
-[Kubernetes version and version skew support policy]: https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions
-[Kubernetes Security and Disclosure Information]: https://kubernetes.io/docs/reference/issues-security/security/#report-a-vulnerability
+Please report security vulnerabilities to: security@servicebinding.io
```
Comment (baijum) [inline SECURITY.md:5]: I kept this PR in draft form as the email address doesn't exist. Let me know if we should use a couple of our contributor's email addresses here.

Comment (scothis) [reply SECURITY.md:5]: Can we use the same emails as the code of conduct?

Comment (baijum) [reply SECURITY.md:5]: Let's check with @nebhale and @arthurdm If they are fine with it. I have added a topic for discussion to our next call.

Comment (baijum) [reply SECURITY.md:5]: Based on the community call discussion, we have decided to create security@servicebinding.io email address and add @scothis and @sadlerap as the recipients. I will ask Red Hat's open source program team to help on the same.

Comment (baijum) [reply SECURITY.md:5]: The email address has been created.
@scothis @sadlerap Please confirm if you have received a test mail from Josh (My colleague in Red Hat).

**Pull request merged**